pkgs/development/libraries/libxslt/default.nix

```
@@ -52,6 +52,8 @@ stdenv.mkDerivation rec {
     "--without-debugger"
   ] ++ lib.optionals pythonSupport [
     "--with-python=${python}"
+  ] ++ lib.optionals (!pythonSupport) [
+    "--without-python"
```

https://github.com/NixOS/nixpkgs/blob/bb961e670f16adf5a9ccae96b82edfd9881fd730/lib/strings.nix#L663-L672

```suggestion
    (lib.withFeatureAs pythonSupport "python" python)
```

---

@OPNA2608 great catch!